src/errors/bad-request-exception.ts

```diff
@@ -0,0 +1,6 @@
+export class BadRequestException extends Error {
+  public readonly code = 400;
+  constructor(message = 'Bad Request') {
```

Comment:
```suggestion
  public readonly code = 400;

  constructor(message = 'Bad Request') {
```
Пустой строки не хватает